toolchain/parse/testdata/array/fail_syntax.carbon

```diff
@@ -0,0 +1,141 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+// --- type.carbon
+
+// CHECK:STDERR: type.carbon:[[@LINE+3]]:12: ERROR: Expected `;` in array type.
+// CHECK:STDERR: var x: [i32];
+// CHECK:STDERR:            ^
+var x: [i32];
+
+// --- invalid_char.carbon
+
+// CHECK:STDERR: colon.carbon:[[@LINE+12]]:12: ERROR: Expected expression.
```

Comment:
Looks like you need to re-autoupdate to match the new filename for this section.

Reply:
Yep, oops.